src/coreclr/gc/gcpriv.h

```diff
@@ -571,9 +580,17 @@ enum allocation_state
     a_state_trigger_ephemeral_gc,
     a_state_trigger_2nd_ephemeral_gc,
     a_state_check_retry_seg,
+    // a temp state I added to distinguish 
+    a_state_wait_in_tams,
```

Comment:
did you want to get rid of this?

Reply:
Yes, this seems low value at this point.